--- xfce4-taskmanager-1.0.0/configure.ac~	2010-06-14 13:02:03.000000000 +0200
+++ xfce4-taskmanager-1.0.0/configure.ac	2010-11-11 20:12:46.184144833 +0100
@@ -149,7 +149,7 @@
 echo
 echo "Build Configuration:"
 echo
-echo "* Wnck:          ${WNCK_FOUND:-no} ${WNCK_VERSION:+(libwnck $WNCK_VERSION)}"
+echo "* Wnck:          ${WNCK_FOUND:-no} ${WNCK_VERSION:+\(libwnck $WNCK_VERSION\)}"
 echo "* Target OS:     $target_os ($ac_os_implementation)"
 echo "* Debug Support: $enable_debug"
 echo
